telosnetwork/telos-wallet
# transaction history fetches data in parallel much faster

# Fixes #445

## Description
This PR splits the process of fetching transaction history data and let them run in parallel. This result in a much faster loading time.

## Test scenarios
- go to [this link](https://deploy-preview-447--wallet-staging.netlify.app/)
- login on EVM with an account with some history
- goto history tab
   - you should see the history page loading gradually and not all at once

## Checklist:
